examples/NER_Plugin/requirements.txt

```diff
@@ -3,4 +3,5 @@ torch==1.8.1+cu111
 -f https://download.pytorch.org/whl/torch_stable.html
 transformers
 sklearn
-rouge-score
\ No newline at end of file
+rouge-score
```

Comment:
@egonzalez125 why don't we make this just install pytorch (which we don't have as a strict requirement) and pymarlin[plugins] which would install the rest here.

Reply:
works for me :)